demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import annotations
+
+import re
+from typing import Any, Dict, Iterable, List, Set, Tuple, Union
+
+from demisto_sdk.commands.common.constants import (
+    RN_HEADER_BY_FILE_TYPE,
+)
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.content_graph.common import (
+    ContentType,
+)
+from demisto_sdk.commands.content_graph.objects.content_item import ContentItem
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+CONTENT_TYPE_SECTION = re.compile(r"^#### ([\w ]+)$\n([\w\W]*?)(?=^#### |\Z)", re.M)
+CONTENT_ITEM_SECTION = re.compile(
+    r"^##### (.+)$\n([\w\W]*?)(?=^##### |\Z)|^- (?:New: )?$", re.M
+)
+
+
+def remove_none_values(ls: Union[List[Any], Tuple[Any, ...]]) -> List[Any]:
+    """
+    Filters out None values from a list or tuple.
+
+    Args:
+        ls (List or Tuple): The list or tuple to filter.
+
+    Returns:
+        List: Filtered list with None values removed.
+    """
+    return list(filter(None, ls))
+
+
+class ReleaseNoteHeaderValidator(BaseValidator[ContentTypes]):
+    error_code = "RN114"
+    description = "Validate the content items mentioned in second-level headers (##), exist in the pack."
+    rationale = "Providing documentation with accurate headers, and avoid confusion."
+    error_message = (
+        "The following headers that were found were invalid:\n"
+        "{content_type_message}\n{content_item_message}\n"
+    )
```

Comment:
```suggestion
    description = "Validate the existence of content types in the first-level headers (####) and the content items in second-level headers (#####)."
    rationale = "Providing documentation with accurate information and avoiding confusion."
    error_message = (
        "The following release note headers are invalid:\n"
        "{content_type_message}\n{content_item_message}\n"
    )
```
correct me if I'm wrong, we don't validate only the content items but also the types